packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -198,8 +199,8 @@ export async function fetchAndParseAsJSONHelper<T>(
         throwOdspNetworkError(
             "errorWhileParsingFetchResponse",
             fetchIncorrectResponse,
-            content,
-            undefined,
+            content, // response
```

Comment:
nit while you're here: Can you just rename content to response? The whole function makes way more sense then.

Reply:
ping

Reply:
Weird, I'm sure I replied, I do not want to touch IOdspResponse<T> as it will impact a long of lines of code. Let's do such rename in separate PR

Reply:
You can do const `{ content: response, headers, headers, commonSpoHeaders, duration } = ...` to preserve `IOdspResponse but give the variable a better name here.

You did mention that in another comment, I just didn't register what you were talking about :P